ports/gamenetworkingsockets/portfile.cmake

```
@@ -1,23 +1,48 @@
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ValveSoftware/GameNetworkingSockets
-    REF 9875f39ad3fae8c913daa310c7ff0a7a54b1e6c2 # v1.3.0
-    SHA512 edf814ef10a11b67045d90d589a3bd6471fa46bc3f16cf1e83d1806df6bb48306dcda4096a2ff80fee78b24ce36d164cd46808680c9f5f8f546455fce7bf9aa0
+    REF 5c793b9f1a507aa9670c8634817bde48b744428b # v1.3.0

```

```suggestion
    REF 5c793b9f1a507aa9670c8634817bde48b744428b
```
We shouldn't mislead the contributors.